e2e/runner/cypress-runner-utils.js

```diff
@@ -30,9 +32,34 @@ function executeYarnCommand({ command, message } = {}) {
   });
 }
 
+const args = arg(
+  {
+    "--folder": String, // The name of the folder to run files from
+    "--open": [Boolean], // Run Cypress in open mode or not? Doesn't accept additional arguments
+  },
+  { permissive: true }, // Passes all other flags and args to the Cypress parser
+);
```

Comment:
Any custom flag that is not defined here will throw an error in Cypress CLI.
That's why I had to extract it here and to use it in both snapshots and main tests, although we will never use neither `--folder` nor `--open` flags for snapshots.